src/react-apps/applications/runtime/src/features/instantiate/containers/InstantiationErrorPage.tsx

```diff
@@ -35,6 +36,7 @@ function InstantiationErrorPage(props: IInstantiationErrorPageProps) {
                 content={content}
                 statusCode={statusCode}
                 titleFontWeight={'medium'}
+                titleColor={AltinnAppTheme.altinnPalette.primary.blueDarker}
```

Comment:
Not really a fan of extending interfaces of a component just for styles, when we have material UI and can use styling (css).

Reply:
Fixed it together